DataManagementSystem/Client/ConsistencyInspector.py

```diff
@@ -235,6 +235,27 @@ def _getTSFiles( self ):
 
     return processedLFNs, nonProcessedLFNs, nonProcessedStatuses
 
+  def __getDirectories( self ):
+    """ get the directories where to look into (they are either given, or taken from the transformation ID
+    """
+    if self.directories:
+      directories = []
+      printout = False
+      for directory in self.directories:
+        if not directory.endswith( '...' ):
+          directories.append( directory )
+        else:
+          printout = True
+          topDir = os.path.dirname( directory )
+          res = self.dm.getCatalogListDirectory( topDir )
```

Comment:
DataManager does not have this method any more. Use FileCatalog() object to get the directory listing